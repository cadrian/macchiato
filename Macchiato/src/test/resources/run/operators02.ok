def val(a) {
    System.Print(System.ToString(a));
    Result = a;
}

def runTest() {
    System.Print(System.ToString(val(4) ^ val(3) ^ val(2)));
}

BEGIN SEQUENCE {
    runTest();
}
