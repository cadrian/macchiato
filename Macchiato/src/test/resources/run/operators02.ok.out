4
3
2
262144
